sos/report/plugins/ceph_mgr.py

```diff
@@ -103,17 +72,66 @@ def setup(self):
             "version"
         ]
 
+        directory = ''
+        if not microceph_pkg:
+            directory = '/var/run/ceph'
+            self.add_file_tags({
+                '/var/log/ceph/(.*/)?ceph-mgr.*.log': 'ceph_mgr_log',
+            })
+
+            self.add_forbidden_path([
+                "/etc/ceph/*keyring*",
+                "/var/lib/ceph/**/*keyring*",
+                "/var/lib/ceph/**/osd*",
+                "/var/lib/ceph/**/mon*",
+                # Excludes temporary ceph-osd mount location like
+                # /var/lib/ceph/tmp/mnt.XXXX from sos collection.
+                "/var/lib/ceph/**/tmp/*mnt*",
+                "/etc/ceph/*bindpass*",
+            ])
+
+            self.add_copy_spec([
+                "/var/log/ceph/**/ceph-mgr*.log",
+                "/var/lib/ceph/**/mgr*",
+                "/var/lib/ceph/**/bootstrap-mgr/",
+                "/run/ceph/**/ceph-mgr*",
+            ])
+
+        else:
+            directory = '/var/snap/microceph'
+            self.add_file_tags({
+                '/var/log/ceph/common/logs/ceph-mgr.*.log': 'ceph_mgr_log',
+            })
+
+            self.add_forbidden_path([
+                "/var/snap/microceph/common/**/*keyring*",
+            ])
+
+            self.add_copy_spec([
+                "/var/snap/microceph/common/logs/ceph-mgr*.log",
```

Comment:
same as above

Reply:
I can do that but that leaves some ones inside the if for "ceph" case and then the common one, so 2 calls result. So leaving it as it is as discussed on MM